### ISSUE TITLE
Transfer/gift holiday/vacation days

### ISSUE THREAD

**Issue by marnovo** [src-d/guide#233]
Do we have any policy on if it is possible for one employee to gift/transfer a number of their currently available holiday/vacation days to another fellow employee? E.g.:
> Person X has 20 days of holidays available to take, then gives 2 days to person Y which has 0, thus in the end X has 18 and Y has 2 days.

**Comment by estherrgarcia** [comment]
No, we don't have it. The law says that the holidays are a right of the employee and the employee has to generate these days.
The company can decide to do whatever the company wants (as the same way the company decided that you can work during the weekend and get an extra day), but always if every part agrees (and signing an agreement with the information).
In my opinion, it doesn't have any sense, the company gives freedom enough to every employee to be able to manage their holidays, no one needs to trade with days. 
@eiso & @jorgeschnura if you want to think about that, the logistics for me are not difficult, we only have to prepare a document and change the days in KIN everytime someone wants to do it. 
@marnovo you can test it with me :laughing:

**Comment by eiso** [comment]
I think it's something that is nice and kind-hearted, matches our culture, and I don't see any downside to it. It has my OK. I would like @jorgeschnura's input before proceeding, but if he's OK with it, @estherrgarcia can you prepare the PR to the guide for it.

**Comment by tsolakoua** [comment]
I agree with @estherrgarcia as I cannot think of any use cases, but it's true it's kind-hearted without any downsides. 

As the holidays are paid, I have a question; if Alice receives gift vacation days from Bob, does that mean that the company will compensate more/less if Alice leaves without having used the days (because maybe Alice and Bob have different salary levels)?

**Comment by eiso** [comment]
@tsolakoua you raise a good point, these days could not be compensated accordingly.

**Comment by jorgeschnura** [comment]
But they should, otherwise the company is saving money without having done any merit for it

**Comment by eiso** [comment]
Good point. @jorgeschnura can we compensate them at their original value? (_insert blockchain joke_)

**Comment by smola** [comment]
The problem here is that, AFAIK, no employee can legally agree to give up their minimum holidays without receiving an additional economic compensation.

Even if these are traded, the company would still be obligated to pay to the one who traded them off.

**Comment by eiso** [comment]
That's good to know @smola, I'd still be okay with that since the people getting let go is rare and a fair use policy on trading should solve most issues.
